<launch>

  <!-- Load image params -->
  <rosparam file="$(find tuber-selection)/src/config/calibration_params.yaml" command="load" ns="calibration"></rosparam>

  <!-- Camera node -->
  <node pkg="cv_camera" name="cv_camera" type="cv_camera_node" output="screen">
    <param name="device_id" value="2" />
    <param name="image_width" value="1280" />
    <param name="image_height" value="720" />
    <param name="rate" value="5" />
    <!-- TODO add camera calibration file -->
    <!-- <param name="camera_info_url" value="$(find tuber-selection)/src/config/camera_info.yaml"></param> -->
  </node>

  <!-- CLI node -->
  <node pkg="tuber-selection" name="cli_node" type="cli.py" output="screen"></node>

</launch>